src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1090,7 +1130,7 @@ private static bool IsTypeParameterDisallowingAnnotation(TypeSymbol typeOpt)
         public override BoundNode VisitLocal(BoundLocal node)
         {
             var local = node.LocalSymbol;
-            int slot = GetOrCreateSlot(local);
+            int slot = GetOrCreateSlot(local, createIfMissing: false);
```

Comment:
>createIfMissing: false [](start = 46, length = 22)

I am not sure if passing this value is going to have any useful effect. If we are not interested in allocating new slots, perhaps we should use VariableSlot API instead.